integration_tests/integration/incidentDetailsSubmittedEdit.cy.ts

```diff
@@ -141,12 +141,14 @@ context('Incident details (edit after completion of report)', () => {
     incidentDetailsPage.reportingOfficerLabel().should('contain.text', 'Reporting officer')
     incidentDetailsPage.reportingOfficerName().should('contain.text', 'USER ONE')
   })
-  it('should show error if one of the time fields is not filled in correctly', () => {
+
+  it('should show error if one of the time fields is not filled in correctly - 3', () => {
     cy.visit(adjudicationUrls.incidentDetails.urls.submittedEdit('G6415GD', 34))
     const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
     incidentDetailsPage.timeInputHours().clear()
     incidentDetailsPage.timeInputMinutes().clear()
     incidentDetailsPage.timeInputHours().type('13')
+    incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').click()
```

Comment:
Same as in the other file - As this is an edit page, you should not need to select either of the yes or no radio buttons, the correct one should already be checked (the controller reads the date of discovery data on the draft adjudication which is retrieved via an API call). I think the reason these tests were failing is because you haven't added the date of discovery data into the stub for the 'stubGetDraftAdjudication' and 'stubEditDraftIncidentDetails' calls.